### PR TITLE
Add example showcasing DIGEST-SHA-512-256 configuration

### PR DESCRIPTION
**Pull request by Skyllarr**

### CODE REVIEW

**Comment by Skyllarr** [comment]
@fjuma This can be reviewed and merged now

File: simple-webapp-digest-algorithms/pom.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <packaging>war</packaging>
+    <version>2.0.0.Alpha1-SNAPSHOT</version>
+    <name>simple-webapp using DIGEST-SHA256 and DIGEST SHA-512-256</name>
+    <url>http://maven.apache.org</url>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <version.wildfly.maven.plugin>2.0.0.Final</version.wildfly.maven.plugin>
+        <version.server.bom>27.0.1.Final</version.server.bom>
```
Comment (fjuma) [inline simple-webapp-digest-algorithms/pom.xml:15]: Can update to 28.0.0.Final now

Comment (Skyllarr) [reply simple-webapp-digest-algorithms/pom.xml]: @fjuma Updated

**Review by fjuma — APPROVED** [review]
Thanks @Skyllarr!

**Pull request merged**